eng/testing/tests.mobile.targets

```diff
@@ -52,13 +52,13 @@
              AssemblyFile="$(RuntimeConfigParserTasksAssemblyPath)"
              Condition="'$(RuntimeConfigParserTasksAssemblyPath)' != ''" />
 
-  <Target Name="GenerateRuntimeConfig">
+  <Target Name="GenerateRuntimeConfig" Condition="'$(OutputType)' != 'library'">
```

Comment:
This shouldn't be needed for browser, correct? If so, then can we add !browser to the condition? I know the target is being used for non-browser cases, but adding to the condition would prevent it from getting used by mistake.

Reply:
Added condition.